editor/plugins/lightmap_gi_editor_plugin.cpp

```diff
@@ -166,19 +169,30 @@ void LightmapGIEditorPlugin::bake_func_end(uint64_t p_time_started) {
 }
 
 void LightmapGIEditorPlugin::_bind_methods() {
-	ClassDB::bind_method("_bake", &LightmapGIEditorPlugin::_bake);
+	ClassDB::bind_method("_bake", &LightmapGIEditorPlugin::_bake, DEFVAL(false));
 }
 
 LightmapGIEditorPlugin::LightmapGIEditorPlugin() {
+	bake_preview = memnew(Button);
+	bake_preview->set_theme_type_variation("FlatButton");
+	bake_preview->set_icon(EditorNode::get_singleton()->get_editor_theme()->get_icon(SNAME("BakePreview"), EditorStringName(EditorIcons)));
+	bake_preview->set_tooltip_text(TTR("Bakes lightmaps with low-quality settings for quick iteration.\nPreview bake quality can be changed in the Rendering > Lightmapping > Preview Bake section of the Project Settings."));
+	bake_preview->set_text(TTR("Preview Bake"));
+	bake_preview->hide();
+	bake_preview->connect("pressed", callable_mp(this, &LightmapGIEditorPlugin::_bake).bind(true));
+	add_control_to_container(CONTAINER_SPATIAL_EDITOR_MENU, bake_preview);
+
 	bake = memnew(Button);
 	bake->set_theme_type_variation("FlatButton");
 	// TODO: Rework this as a dedicated toolbar control so we can hook into theme changes and update it
 	// when the editor theme updates.
 	bake->set_icon(EditorNode::get_singleton()->get_editor_theme()->get_icon(SNAME("Bake"), EditorStringName(EditorIcons)));
+	bake->set_tooltip_text(TTR("Bakes lightmaps with the settings specified in the LightmapGI node."));
 	bake->set_text(TTR("Bake Lightmaps"));
 	bake->hide();
 	bake->connect("pressed", Callable(this, "_bake"));
```

Comment:
So what exactly happens when you change it to `callable_mp`? There is no reason to keep using strings and keep binding this method.

Reply:
If I use `callable_mp()` for the method without `.bind(false)` (the parameter is optional), I get an error when the editor opens and clicking the button does nothing.

I suppose I could always use `.bind(false)` or make the argument required though.

Reply:
I think you should just bind false.